symplyphysics/laws/kinematic/accelerated_velocity_from_time.py

```diff
@@ -0,0 +1,26 @@
+from symplyphysics import (
+    symbols, Eq, pretty, solve, Quantity, units,
+    validate_input, validate_output, expr_to_quantity
+)
+
+# Description
```

Comment:
Better to add new laws in separate PRs. I suggest to extract new laws to separate branch and remove them from here.